dpdata/lammps/dump.py

```diff
@@ -81,8 +82,11 @@ def safe_get_posi(lines,cell,orig=np.zeros(3)) :
     posis.sort()
     posis = np.array(posis)[:,1:4]
     if not sf:
-        posis = (posis-orig)@np.linalg.inv(cell)# Convert to scaled coordinates for unscaled coordinates
-    return (posis%1)@cell # Convert scaled coordinates back to Cartesien coordinates 
+        posis = (posis - orig) @ np.linalg.inv(cell)  # Convert to scaled coordinates for unscaled coordinates
+    if uw and unwrap:
+        return posis @ cell  # convert scaled coordinates back to Cartesien coordinates unwrap at the periodic boundaries
+    else:
+        return (posis % 1) @ cell  # Convert scaled coordinates back to Cartesien coordinates with wraping at periodic boundary conditions
```

Comment:
throw a warning if uw and not unwrap?

Reply:
Because safe_get_posi will be called for each frame of the dump, put a warning here will throw warnings multiple times. I guess it might be better to only throw  warning once after reading them all in system_data(), but over there it does not have uw.  

What would be the preferred way to log the warining?

Reply:
https://docs.python.org/3/library/warnings.html#the-warnings-filter

see "once"

Reply:
Added.